root-parent-pom/pom.xml

```diff
@@ -377,6 +377,9 @@
         </profile>
         <profile>
             <id>java11</id>
+            <activation>
+                <jdk>11</jdk>
```

Comment:
I am not sure but should we not use something like "[11," instead?

Reply:
agree

Reply:
that matches the existing build config